components/templates/ArticleTemplate/ArticleTemplateStyles.ts

```diff
@@ -75,6 +75,11 @@ export const StyledMarkdownWrapper = styled.div<Props>`
     a {
         color: var(--primary-link-color);
         text-decoration: none;
+
+        :hover {
+            color: ${props => props.isDark
+                ? '#5033e1'
+                : '#bfbfff'};
```

Comment:
We could also do `#212529` (default text color for light) in light mode. Not sure what's better. I realize the lighter shade may not be great for contrast. The next shade "down" in the brand scheme (and the darkest shade of blurple) is `3826cc` which doesn't seem different enough.

Reply:
I think this looks good as it is